### PR TITLE
Ramone/forward

### PR DESCRIPTION
**Pull request by Ramone-HL**
Added Forward Transfer chapter

- added image files
- updated sequence order in connect folder
- updated sidebar

### CODE REVIEW

**Review by yurikoinaba — COMMENTED** [review]
Looks good. Can we make the images for the full page Sphere screenshots larger? It's hard to read the details of the screenshot

**Comment by Ramone-HL** [comment]
Increased the image size for Block Explorer in Step 8.

**Comment by yurikoinaba** [comment]
Thansk @Ramone-HL . Let's also do the same for steps 3 and 7

**Review by yurikoinaba — APPROVED** [review]

File: static/img/docs/forward/screenshot.png
Comment (rabyyuson) [inline static/img/docs/forward/screenshot.png:1]: @Ramone-HL is this needed or was it part of a test? If it's not needed can we remove?
It's appearing as follows:

Screenshot 2023-10-23 at 3 50 24 PM

**Review by rabyyuson — CHANGES_REQUESTED** [review]

File: static/img/docs/forward/screenshot.png
Comment (Ramone-HL) [reply static/img/docs/forward/screenshot.png:1]: deleted screenshot.png...not needed

**Review by rabyyuson — APPROVED** [review]

**Pull request merged**